html/src/components/terminal/xterm/index.ts

```diff
@@ -386,6 +388,19 @@ export class Xterm {
                 case 'isWindows':
                     if (value) console.log('[ttyd] is windows');
                     break;
+                case 'unicodeVersion':
+                    switch (value.toString()) {
```

Comment:
```suggestion
                    switch (value) {
```

Reply:
This seemed necessary when I used the CLI `--client-option "unicodeVersion=6"`. Seemed the value was being cast as a number and therefore didn't match the switch. This was the fix for this problem. Is it possible I missed something?

Reply:
Just retested my changes and can confirm this is necessary. My knowledge of JS is fairly low, so this may be caused by another reason?

Reply:
The value in `-t key=value` is parsed as json value. To get a string value of `6`, you may use `-t unicodeVersion='"6"'`.

To make `-t unicodeVersion=6` work, you can add another `case 6:` and `case 11:` for number value.

Reply:
`value.toString()` will not work for null value. js switch works for all types including null.

Reply:
Thanks for the explanation.

Converting it to a string seems safe then. User doesn't need to care if they pass it as a number or string, we just internally handle it.

Adding extra checks for strings AND numbers seems wasteful.

Reply:
Obviously your decision though, let me know whether you still want me to make the change or leave it as it is.

Reply:
> `value.toString()` will not work for null value. js switch works for all types including null.

Missed this reply, making the changes as you requested. Thanks for the explanation.